store/postgres/migrations/2019-04-19-171709_split_entities/up.sql

```diff
@@ -0,0 +1,106 @@
+-- metadata to track the DB schemas we create for subgraph deployments
+
+-- The name of the DB schema corresponding to a deployment
+create table sgd_schemas(
```

Comment:
Yep, I renamed it. We should have a longer convo about code readability at some point since we'll keep going back and forth on these. My basic point is that what makes code hard to read are concepts that are not well explained (and I definitely should have put more comments on the `SplitTable` struct to make it easier to understand what that's for) Anyway, I changed the name of the table.

Reply:
I agree. There's an amount of ambiguity in the terms we use (like "subgraph" is being used for a bunch of things). I don't feel like introducing acronyms is the answer though. Let's discuss at some point! 😄